### ISSUE TITLE
Build Issue on Windows

### ISSUE THREAD

**Issue by hohilwik** [SebLague/Tiny-Chess-Godot#1]
I have been trying to build this project on Windows, but while it does export with a few warnings, I only see the default graphics and none of the button titles or scripts load.

I wanted to ask if this has been tested on Windows, or is even finished? Or is it an issue on my end?

Here is what the exported EXE looks like, the buttons click but don't do anything:

![image](https://github.com/SebLague/Tiny-Chess-Godot/assets/80057848/58beab11-4814-40f9-9ed5-521ba3823b35)

**Comment by SebLague** [comment]
Yes it's finished and was tested on Windows 10. I recommend making a debug build and then running that. You should get some sort of error message in the console when running. If you post it here I can try figure out what's going on.
